test/cmdlineTests/function_debug_info/input.sol

```diff
@@ -0,0 +1,6 @@
+contract C {
+  function f(uint[] calldata x) pure external returns (uint) { return x[0]; }
+  // This will be optimized out
+  function g() pure internal {}
+  mapping(uint => uint) public t;
```

Comment:
```suggestion
    function f(uint[] calldata x) pure external returns (uint) { return x[0]; }
    // This will be optimized out
    function g() pure internal {}
    mapping(uint => uint) public t;
```

Same in the other test.